obfuscator.py

```diff
@@ -52,14 +54,19 @@
 from functools import cache, reduce
 from math import log2, trunc
 
-"""
+version = version_info.major + version_info.minor / 100
+if version < 3.11:
+    print("\nThis obfuscator was primarily written for Python3.11, and some things may not work correctly in earlier versions.\n")
+    warn(FutureWarning("use 3.11 or stuff might not work"))
+elif version > 3.11:
```

Comment:
My rational for checking that because 3.12 prerelease versions *are* out there, and some people might use them. Having looked over the whats new section, it doesn't look like that section is necessary, however. I'll go ahead and remove it.